src/components/GroupsTable/GroupsTable.js

```diff
@@ -208,6 +246,47 @@ const GroupsTable = () => {
                 }}
                 filterConfig={{ items: filterConfigItems }}
                 activeFiltersConfig={activeFiltersConfig}
+                bulkSelect={{
+                    items: [
```

Comment:
A minor: I think if we move this bulk select config object outside of the JSX rendering section, we would have even cleaner code. We can also separate this into a callback as the function definition does not change often.